paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentElementLoader.kt

```diff
@@ -294,16 +294,14 @@ internal class DefaultPaymentElementLoader @Inject constructor(
         state: PaymentElementLoader.State
     ) {
         logLinkHoldbackExperiment(
```

Comment:
Instead of re-triggering the experiment logic multiple times, we now compute the dimensions once and send the multiple exposures. This prevents multiple lookup calls per experiment.